pkg/dao/helpers.go

```diff
@@ -112,10 +113,10 @@ func isOwnedRepository(db *gorm.DB, orgID string, repositoryConfigUUID string) (
 	return true, nil
 }
 
-func checkForValidRepoUuidsUrls(uuids []string, urls []string, db *gorm.DB) (bool, bool, string, string) {
+func checkForValidRepoUuidsUrls(ctx context.Context, uuids []string, urls []string, db *gorm.DB) (bool, bool, string, string) {
```

Comment:
would checkForValidSnapshotUuids in this file also need a context passed in?

Reply:
good catch! i missed those when doing a rebase.  Will update

Reply:
fixing checkForValidSnapshotUuids, but isOwnedRepository actually passes in the DB, so in that case i think its okay to presume that the db is passed in with the context already, at least that is a pattern i used for some other things.